keyboards/redox_media/keyboard.json

```diff
@@ -36,8 +36,7 @@
             "pin": "D0"
         }
     },
-    "processor": "atmega32u4",
-    "bootloader": "caterina",
+    "development_board": "promicro",
```

Comment:
Not for this PR but this appears to be a handwired board. https://shiftux.org/making_projects/keyboard.html